lib/dashboard/grouped_organizations.rb

```diff
@@ -30,6 +30,9 @@ def collect_grouped_options
       groups = @organizations.group_by(&:type)
       options = ["Institution", "Provider", "Program", "Core"].map do |type|
         next unless groups[type].present?
+
+        # Sort organizations by name
+        groups[type].sort! { |lhs, rhs| lhs.name <=> rhs.name }
```

Comment:
I wish @organizations was an AR relation, but the ProtocolsController passes in an Array of Organizations rather than an AR relation. But we can still sort before the group_by! Better to sort once than up to four times.